upgrading/automated_upgrades.adoc

```diff
@@ -192,11 +192,14 @@ changes you made, apply the equivalent appropriate changes to your inventory
 files before running the upgrade. Otherwise, your manual changes might be
 overwritten by default values during the upgrade, which could cause pods to not
 run properly or other cluster stability issues.
-.. To check if your certificates will expire, set the
+.. To check if your certificates will expire, set
+`openshift_certificate_expiry_fail_on_warn` to `True` and the
```

Comment:
We don't need this explain IMHO. This parameter default to be True.

Reply:
@gpei, if they're upgrading from 3.10, would that parameter be in the file?